README.md

```diff
@@ -1,14 +1,66 @@
 # Auth2 client for Java
 
 This repo contains a minimal client for the [KBase Auth2 server](https://github.com/kbase/auth2),
-covering only the most common operations - e.g. validating tokens and user names.
+covering only the most common operations - e.g. validating tokens and user names. For those
+functions, it is backwards compatible with the prior version of the client.
 
 Most other uses are easily done with any http/REST client like the built in Java client
 in 11+ or the Jersey client.
 
+## Including the client in your build
+
+See https://jitpack.io/#kbase/auth2_client_java for instructions on how to include JitPack
+built dependencies in your build.
+
+## JavaDoc
+
+JavaDoc is available at
+```
+https://javadoc.jitpack.io/com/github/kbase/auth2_client_java/<version>/javadoc/
+```
+
+For example:
+
+https://javadoc.jitpack.io/com/github/kbase/auth2_client_java/0.5.0/javadoc/
+
+## Usage
+
+If backwards compatibility with old code is required, use the
```

Comment:
How old? Got a date estimate? Or other way of saying how old?

Reply:
Fixed